README.adoc

```diff
@@ -197,3 +220,94 @@ ath: "default"
 ----
 
 *Please note that a blank metadata file will result in an error*
+
+=== runPCT
+
+Runs the link:https://github.com/jenkinsci/plugin-compat-tester[Plugin Compat Tester] in a configurable way.
+
+The configuration is divided into two parts, one related to the step itself and another related to how the PCT is run.
+To configure the step just use the step's parameters described below, to configure the PCT runs a metadata file (in YAML format)
+is used. Further sections describe the metadata file in detail. *Note that if the metadata file does not exist this step
+will do nothing at all.*
+
+The list of step's params and the related default values are:
+
+
+`pctUrl`::
```

Comment:
Would be nice to support passing custom PCT CLI options. I need it for https://issues.jenkins-ci.org/browse/INFRA-1568, but I can implement it on my own after the merge